### PR TITLE
Yoshi Server: error toast message

### PR DESCRIPTION
**Pull request by yanivefraim**
We want to display an informative error modal in case of a server function error:

Kapture 2020-05-20 at 17 16 34

### CODE REVIEW

File: packages/yoshi-server-client/src/index.ts [hunk 1]
```diff
@@ -56,9 +56,11 @@ export default class implements HttpClient {
         if (res.headers.get('content-type')?.includes('application/json')) {
           const error = await res.json();
           if (process.env.NODE_ENV !== 'production') {
+            if (process.env.BROWSER !== 'none') {
```
Comment (ronami) [inline packages/yoshi-server-client/src/index.ts:59]: I think it's process.env.browser (lowercase): https://github.com/wix/yoshi/blob/master/packages/yoshi-common/src/webpack.config.ts#L735.

File: packages/yoshi-server-client/src/utils.ts [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
   return baseUrl.replace(/\/+$/, '') + '/' + relativeUrl.replace(/^\/+/, '');
 }
+
+export function showToast<
+  Result extends FunctionResult,
+  Args extends FunctionArgs
+>(error: any, method: DSL<Result, Args>, args: any, baseUrl: string) {
+  const Toastify = require('toastify-js');
+  const css = createCss();
+  document.body.appendChild(css);
```
Comment (ronami) [inline packages/yoshi-server-client/src/utils.ts:16]: can we append css to the document just once? I think it's the same CSS for all errors/toasts.

File: packages/yoshi-server-client/src/index.ts (hunk 1 above)
Comment (yanivefraim) [reply packages/yoshi-server-client/src/index.ts]: Thanks, good catch

File: packages/yoshi-server-client/src/utils.ts (hunk 2 above)
Comment (yanivefraim) [reply packages/yoshi-server-client/src/utils.ts]: sure, fixed

File: packages/yoshi-server-client/src/utils.ts [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+          credentials: 'same-origin',
+          method: 'GET',
+        },
+      );
+    },
+  }).showToast();
+}
+
+function createCss() {
+  const toastCloseClass = '.toast-close';
```
Comment (ronami) [inline packages/yoshi-server-client/src/utils.ts:46]: I think this is a mistake

Comment (yanivefraim) [reply packages/yoshi-server-client/src/utils.ts]: fixed

**Pull request merged**